makeflow/src/makeflow.c

```diff
@@ -2153,7 +2160,11 @@ int main(int argc, char *argv[])
 	}
 
 	printf("starting workflow....\n");
-	makeflow_hook_dag_start(d);
+	rc = makeflow_hook_dag_start(d);
+	if(rc != MAKEFLOW_HOOK_SUCCESS){
+		debug(D_ERROR, "Failed dag start hooks");
```

Comment:
Stylistically, do we want to write DAG in all caps in the error message for clarity? This may not matter since we can assume if a user is reading through the debug log, they probably know what a DAG is.